tests/Tests/100__deploy/110__auth_providers_and_rbac/110__rbac.robot

```diff
@@ -0,0 +1,41 @@
+*** Settings ***
+Resource    ../../../Resources/ODS.robot
+Resource    ../../../Resources/Page/ODH/ODHDashboard/ODHDashboard.resource
+
+
+*** Test Cases ***
+Any OpenShift User Should Also Be A RHODS User
+    [Documentation]    Any OpenShift user should also be a RHODS user
+    [Tags]    Sanity
+    ...       ODS-1164
+    Set Library Search Order    SeleniumLibrary
+    Open Browser    ${OCP_CONSOLE_URL}    ${BROWSER.NAME}    options=${BROWSER.OPTIONS}
+    Login To Openshift    ${OCP_ADMIN_USER.USERNAME}    ${OCP_ADMIN_USER.PASSWORD}    ${OCP_ADMIN_USER.AUTH_TYPE}
+    Go To Config-Maps
+    Select Redhat-ODS-Applications
+    Select Rhods-Groups-Config
+    Wait Until Page Contains Element    xpath=//*[@class='co-pre-wrap co-copy-to-clipboard__text']
+    Element Text Should Be    xpath=//*[@id="content-scrollable"]//dd[1]/div/pre    dedicated-admins
+    Element Text Should Be    xpath=//*[@id="content-scrollable"]//dd[2]/div/pre    system:authenticated
+    Begin Web Test
```

Comment:
it's the last keyword of the test case but it is named `Begin`, I would change it. 

`Begin Web Test` doesn't spawn a notebook. Do you need to spawn a notebook? I'm asking because ODS-1164 says to do that